plugins/processors/t128_filter/t128_filter.go

```diff
@@ -10,20 +14,36 @@ const sampleConfig = `
   ## The conditions that must be met to pass a metric through. This is similar
   ## behavior to a tagpass, but the multiple tags are ANDed
   [[processors.t128_filter.condition]]
+    ## Mode dictates how to match the condition's tag values
+	## Valid values are:
+	##  * "exact": exact string comparison
+	##  * "glob": go flavored glob comparison (see https://github.com/gobwas/glob)
+	##  * "regex": go flavored regex comparison
+    # mode = "exact"
 
   [processors.t128_filter.condition.tags]
-     #tag1 = ["value1", "value2"]
-	 #tag2 = ["value3"]
+     # tag1 = ["value1", "value2"]
+	 # tag2 = ["value3"]
```

Comment:
intentional indent?

Reply:
Weird, these indents looked fine in vscode before I pushed... I'll fix

Reply:
probably tabs vs. spaces